wolfcrypt/src/asn.c

```diff
@@ -1033,6 +1033,7 @@ static int GetASN_Integer(const byte* input, word32 idx, int length,
         return ASN_PARSE_E;
     }
 #endif
+#ifndef WOLFSSL_ASN_INT_LEAD_0_ANY
     if (input[idx] == 0) {
         /* Check leading zero byte required. */
         if ((length > 1) && ((input[idx + 1] & 0x80) == 0)) {
```

Comment:
Can we move the debug message to not be guarded by WOLFSSL_DEBUG_ASN_TEMPLATE here so that with just using --enable-debug the log message of 'error, unexpected leading 0 found : use WOLFSSL_ASN_INT_LEAD_0_ANY to workaround' is displayed.

Reply:
Done.
Detects issue and debug statement will always happen.